src/global/ValueId.h

```diff
@@ -27,6 +27,7 @@ enum struct Datatype {
   VocabIndex,
   LocalVocabIndex,
   TextRecordIndex,
+  WordVocabIndex,
   MaxValue = TextRecordIndex
```

Comment:
You have to read the note:
The `MaxValue` must always be the last one. 
So either put in `WordVocabIndex` before `TextRecordIndex` or change the `MaxValue` to `WordVocabIndex`.